gitbutler-ui/src/lib/components/Icon.svelte

```diff
@@ -9,6 +9,9 @@
 	export let color: IconColor = undefined;
 	export let opacity: number | undefined = 1;
 	export let spinnerRadius: number | undefined = 5;
+	export let size = 16;
+
+	const pxToRem = (px: number) => `${px / 16}rem`;
```

Comment:
Could you test this on different zoom levels? I think it'll give you an incorrect result.

This is how I've done it in other places, but it's a bit tedious. Maybe we could put this conversion function into `userSettings.ts`?
```js
const pxToRem = (px: number) => `${px / (16 * $userSettings.zoom)}rem`; 
```